boards/APU/results.csv

```diff
@@ -0,0 +1,29 @@
+Test name,APU2C
```

Comment:
This file needs to be recreated with renamed tests.

Also, I find it unlikely that `SKL boot  works,PASS` and ``PCR 17 doesn't contain only `0`s or only `F`s,FAIL`` at the same time.

Reply:
In the test case it's:
Slaunch boot (`slaunch-boot` menuentry) works - TrenchBoot boots into OS

Only checks that system booted from `slaunch-boot` menuentry.

Reply:
Fixed in: [371a7c5](https://github.com/TrenchBoot/test-results/pull/1/commits/371a7c522b90b1eeba4e86cabf2f71db202a3398)
Changed test name to "`slaunch-boot` menuentry boots into OS" and "`boot` menuentry boots into OS"